scripts/detect_bad_deps.js

```diff
@@ -41,7 +41,7 @@ exec("flow check --profile", function(error, stdout, stderr) {
     process.exit(1);
   }
   console.log("Biggest cycle: " + cycle_len);
-  let MAX_CYCLE_LEN = 56;
+  let MAX_CYCLE_LEN = 57;
```

Comment:
The main thing is to explain why. I do wonder, however, if the code will not end up being more readable if FunctionValue and all of its subclasses are kept in the same module.